package.json

```diff
@@ -90,7 +90,6 @@
     "debug": "2.2.0",
     "minimist": "1.2.0",
     "ramda": "0.21.0",
-    "rollup": "0.37.0",
```

Comment:
@snuggs As I said earlier the Rollem it is a "tool for Rollup". So, I can not even imagine people would try to use Rollem without Rollup - it is useless. Thus any dependencies on Rollup are obvious and no need to put them in package.json, imho.